sensing_launch/launch/aip_customized/imu.launch.xml

```diff
@@ -14,7 +16,7 @@
 
 
     <!-- Relay -->
-    <node pkg="topic_tools" type="relay" name="relay" args="/sensing/imu/tamagawa/imu_raw /sensing/imu/imu_data" />
+    <node pkg="topic_tools" exec="relay" name="relay" args="/sensing/imu/tamagawa/imu_raw /sensing/imu/imu_data" />
```

Comment:
This must be replaced with:
```suggestion
   <node pkg="topic_tools" exec="relay" name="twist_relay" output="log" >
     <param name="input_topic" value="/sensing/imu/tamagawa/imu_raw" />
     <param name="output_topic" value="/sensing/imu/imu_data" />
     <param name="type" value="sensor_msgs/msg/Imu" />
   </node>
```

Reply:
Same goes for other imu.launch.xml

Reply:
I copied this to all identical configurations; i.e., all except `xx1` where I added a bit more now that I understand how to invoke the `relay` exec properly. @mitsudome-r please check the changes to `camera.launch.xml` in cc23b23a0226dc702b2658dbdfd1e5c4decfa0f5

Reply:
I mean this change https://github.com/tier4/autoware_launcher.iv.universe/commit/cc23b23a0226dc702b2658dbdfd1e5c4decfa0f5#diff-b99dcce7f16cb67f79b0ab832faa761b28f63d4542ad711f28a31ff03f4c6907R13